src/features/homeFeature/skeletons/CarouselSkeleton.tsx

```diff
@@ -0,0 +1,33 @@
+const CarouselSkeleton = () => {
+  return (
+    <div className="carousel carousel-center p-4 space-x-4 h-[340px] flex gap-2">
+      <div className="carousel-item h-[340px] w-40 flex flex-col gap-2">
+        <div className="skeleton h-64 w-full"></div>
+        <div className="skeleton h-4 w-full"></div>
+        <div className="skeleton h-3 w-14"></div>
+      </div>
+      <div className="carousel-item h-[340px] w-40 flex flex-col gap-2">
+        <div className="skeleton h-64 w-full"></div>
+        <div className="skeleton h-4 w-full"></div>
+        <div className="skeleton h-3 w-14"></div>
+      </div>
+      <div className="carousel-item h-[340px] w-40 flex flex-col gap-2">
+        <div className="skeleton h-64 w-full"></div>
+        <div className="skeleton h-4 w-full"></div>
+        <div className="skeleton h-3 w-14"></div>
+      </div>
+      <div className="carousel-item h-[340px] w-40 flex flex-col gap-2">
+        <div className="skeleton h-64 w-full"></div>
+        <div className="skeleton h-4 w-full"></div>
+        <div className="skeleton h-3 w-14"></div>
+      </div>
+      <div className="carousel-item h-[340px] w-40 flex flex-col gap-2">
+        <div className="skeleton h-64 w-full"></div>
+        <div className="skeleton h-4 w-full"></div>
+        <div className="skeleton h-3 w-14"></div>
+      </div>
+    </div>
+  );
+};
+
+export default CarouselSkeleton;
```

Comment:
Consider using a custom style for the height here instead of an arbitrary value. Either way is fine, just a suggestion.

Reply:
Thank you, noted!👍